src/test/role/__snapshots__/role.test.js.snap

```diff
@@ -609,16 +609,4 @@ Array [
 ]
 `;
 
-exports[`role should render top toolbar 1`] = `
```

Comment:
Looks like this test is broken, can you change in `role.test.js` finding of `.top-toolbar` to `expect(toJson(wrapper.find('TopToolbar'), { mode: 'shallow' })).toMatchSnapshot();`